specs/Terminate Spec.md

```diff
@@ -0,0 +1,53 @@
+# **Terminate Program Spec**
+
+![MS PowerToys](https://hothardware.com/ContentImages/NewsItem/48038/content/Microsoft_PowerToys.jpg "PowerToys")
+## Shortcut for users to quickly terminate a running program
+### Authors: Benjamin Leverette and Prudence Phillips
+### Spec Status: Draft
+> ## 1. Overview
+&nbsp; &nbsp; &nbsp; &nbsp; **1.1. Elevator Pitch / Narrative** 
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  Mike is debugging his code in Visual Studio and the program freezes with the "App not responding" text displaying on the title bar, informing him that he cannot continue his work without closing the program and restarting it. Mike can completely shutdown the app quicker using a Keyboard shortcut if he has this PowerToy installed on his computer.
+
+&nbsp; &nbsp; &nbsp; &nbsp; **1.2. Customers**
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; PowerToys is mainly targeted towards Windows Power Users though it is available to users who want to experience using windows in a more efficient and productive way.
+ 
+  
+&nbsp; &nbsp; &nbsp; &nbsp; **1.3. Problem Statement and Supporting Customer Insights**
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Windows users need a quicker, more efficient way to completely shut down an app whenever it is unresponsive. The team is still required to find a proper name for this PowerToy.
+
+&nbsp; &nbsp; &nbsp; &nbsp; **1.4. Existing Solutions or Expectations**
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Users can already acheive the function of this PowerToy by accessing clicking End Task through Task Manager on Windows. Users need to be running Windows 10 and have PowerToys installed to use it.
+
+
+&nbsp; &nbsp; &nbsp; &nbsp; **1.5. Goals/Non-Goals**
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; - Develop this PowerToy and have sufficient time for testing and integration within our assigned 8 weeks for the project. 
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; - Come up with a suitable name for this PowerToy, preferrably one accepted by the community.
+
+> ## 2. Definition of Success
+&nbsp; &nbsp; &nbsp; &nbsp; **2.1. Expected Impact: Customer, and Technology Outcomes, Experiments + Measures**
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; There is currently a sizable number of people who are following the PowerToys project. We hope to meet these people's expectations and receive good feedback from the community when we are done with this PowerToy.
+
+> ## 3. Requirements
+&nbsp; &nbsp; &nbsp; &nbsp; **3.1.	Functional Requirements**
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; - A keyboard shortcut to morph the mouse pointer into an alternate cursor which when clicked, prompts the user about shutting down the program. 
```

Comment:
What will that alternate cursor look like? What does the prompt look like? What shortcut(s) are used?